libp2p/crypto/crypto.nim

```diff
@@ -70,13 +70,14 @@ when supported(PKScheme.Secp256k1):
 
 import ecnist, bearssl
 import ../protobuf/minprotobuf, ../vbuffer, ../multihash, ../multicodec
-import nimcrypto/[rijndael, twofish, sha2, hash, hmac, utils]
+import nimcrypto/[rijndael, twofish, sha2, hash, hmac]
+import nimcrypto/utils as ncrutils
```

Comment:
let's put a comment here that we use ncrutils because it is constant time (else it's easy to forget and switch to byteutils)

Reply:
done